code/modules/awaymissions/corpse.dm

```diff
@@ -281,9 +282,10 @@
 		//Pre-job equips so Voxes dont die
 		H.dna.species.pre_equip_species_outfit(null, H)
 		H.regenerate_icons()
-		SSquirks.AssignQuirks(H, user.client, TRUE, TRUE, null, FALSE, H)
-		for(var/datum/loadout_item/item as anything in loadout_list_to_datums(H?.client?.prefs?.loadout_list))
-			item.post_equip_item(H.client?.prefs, H)
+		if (!handles_loadout_and_quirks)
+			SSquirks.AssignQuirks(H, user.client, TRUE, TRUE, null, FALSE, H)
+			for(var/datum/loadout_item/item as anything in loadout_list_to_datums(H?.client?.prefs?.loadout_list))
+				item.post_equip_item(H.client?.prefs, H)
```

Comment:
skyrat edit

Reply:
doesn't need one, it's already inside a skyrat edit